packages/@expo/fingerprint/package.json

```
@@ -5,17 +5,21 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "scripts": {
-    "watch": "tsc --watch --preserveWatchOutput",
-    "build": "tsc --sourceMap",
-    "prepare": "yarn run clean && yarn build",
-    "clean": "rimraf build ./tsconfig.tsbuildinfo",
-    "lint": "eslint .",
-    "test": "jest",
-    "test:e2e": "yarn prepare && jest --config e2e/jest.config.js"
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "prepare": "expo-module clean && expo-module build",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "test": "expo-module test",
+    "test:e2e": "expo-module prepare && expo-module test --config e2e/jest.config.js",
+    "typecheck": "expo-module typecheck"
   },
   "bin": {
     "fingerprint": "bin/cli.js"
   },
+  "files": [
+    "build"
```

```suggestion
    "bin",
    "build"
```

we also need the `bin/`